### PR TITLE
Updated README to reflect the Google doc

### PR DESCRIPTION
**Pull request by Bosch-0**
Updated README to reflect the Google doc

### CODE REVIEW

**Comment by pedromvpg** [comment]
This all looks good I think.
@GBKS, @johnsBeharry, we should talk about a PR merge protocol. 
Or did you already discuss this?

**Comment by GBKS** [comment]
Reviewed the changes, look good to me.

We have not talked about a merge protocol in detail yet. Is there some reference material on good practices?

**Comment by moneyball** [comment]
We have discussed a merge protocol. @GBKS you even drafted it :) Step 5 in this doc. https://docs.google.com/document/d/1P20rPq6V6w78dXpBqLiiiB-kSypZOzXaG9ZUErYIq28/edit

I think we try to follow this and then work out any problems through our actual experience.

**Comment by moneyball** [comment]
I would also suggest moving at least the life cycle portion of that doc into a markdown file and put in our GitHub repo, which I think communicates to the community and observers that it is an "official v1" of a process. If we then need to improve the process, Issues or PRs can be submitted against it and a discussion can ensue. Thoughts?

**Comment by GBKS** [comment]
Gotcha, I was just thinking about the actual merge action, not the part about reaching consensus that comes before.

For example, is just clicking the "Merge pull request" button enough? Or should maintainers always post a comment summarizing the consensus and decision to perform the merge.

Also seeing the list of reviewers in the sidebar for the first time. Is that something we want to use?

**Comment by moneyball** [comment]
I think we should use the list of reviewers in GitHub.

It also sounds like a good idea that the maintainer who merges summarizes
the judgment as to why it was merged.

File: README.md [hunk 1]
```diff
@@ -8,14 +8,11 @@ Project Intro
...
 
-
 This is a project overview of a design guide for bitcoin with early thoughts on what it could be.
 
 Like a travel guide, it provides an introduction to a “foreign place”, shows you the highlights and tells you some of the best stories. It provides facts and maps to figure out how to get around, ideas for tours and things to do. Whether you end up taking the official museum tour, clubbing all night or go on shopping sprees is still your choice. Note that these are guides, not mandates. Just like in a foreign city, you can choose to go off the path, and forge a new direction. Others may just follow in the future.
 
-Comments on a previous version have been reviewed and used to improve this doc where relevant.
-
-A lot of discussion about it is taking place in the [bitcoin design Slack](https://bitcoindesign.slack.com/join/shared_invite/zt-fljgbpg0-wHr_thYUYWh5105UnGTWHw#/) (#bitcoin-design-guide).
+Discussion about it is taking place in the [bitcoin design Slack](https://bitcoindesign.slack.com/join/shared_invite/zt-fljgbpg0-wHr_thYUYWh5105UnGTWHw#/) (#bitcoin-design-guide).
```
Comment (johnsBeharry) [inline README.md:15]: The invite link expired, it's no longer valid. The problem seems to be present in the original Google Doc so should not block the merging of this PR.

**Pull request merged**